docs/Reply.md

```diff
@@ -35,36 +35,46 @@ fastify.get('/', {config: {foo: 'bar'}}, function (request, reply) {
 ```
 
 <a name="code"></a>
-### Code
+### .code(statusCode)
 If not set via `reply.code`, the resulting `statusCode` will be `200`.
 
 <a name="header"></a>
-### Header
+### .header(key, value)
 Sets a response header.
```

Comment:
Should we need to document that behaviour?

Reply:
done.